static/panes/cfg-view.ts

```diff
@@ -303,8 +303,10 @@ export class Cfg extends Pane<CfgState> {
 
     override resize() {
         const height = (this.domRoot.height() as number) - (this.topBar.outerHeight(true) ?? 0);
-        this.cfgVisualiser.setSize('100%', height.toString());
-        this.cfgVisualiser.redraw();
+        if ((this.cfgVisualiser as any).canvas !== undefined) {
```

Comment:
I thought this change came in another PR, but it snuck in to this one. This avoids resizing the canvas if it's not actually there, butas it's private, that's why the any is there